include/Pawn.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include "Piece.h"
+#include "macros.h"
+
+class Pawn : public Piece {
+
+public:
+	Pawn(bool);
+	bool isCorrectMove(const Location&, const Location&) const override;
+	Character getCharacter() const override;
+	void setFirstMove();
+
+private:
+	bool m_firstMove = true;
```

Comment:
Is this the correct location for this variable? Was there another place that would be more correct to set it